lib/font-fit.js

```diff
@@ -227,7 +227,7 @@
       }
 
       return {
-        fontFamily: style.fontFamily || 'unknown',
+        fontFamily: heading.style.fontFamily || 'sans-serif',
```

Comment:
but if you don't use the computed style, this will always be empty. I think the issue here is that there are cases where the style resolves to `''`.